wpeverest/everest-forms
# Feature - Akismet integration

### All Submissions:


### Changes proposed in this Pull Request:

This PR is introducing the new anti-spam protection feature ie Akismet. It has two types:

- Make the form submission as failed: Prevent the form submission
- Mark as Spam: Allow to store the entries but the status will the spam and we can view it from entries > spam.

To get the license key -> https://akismet.com/developers/getting-started/

### How to test the changes in this Pull Request:

1. Download the Akismet plugin if you have removed the plugin (note: This plugin comes with a preinstalled while creating a new site)
2. Setup configuration settings ie. license key - b89c0fcf7578
3. Go to form and enable it from form settings > anti-spam and protection tab > enable Akismet protection also verify the warning message.
4. Try to submit the form with valid data and to get an error message test with name = akismet-guaranteed-spam and email = akismet-guaranteed-spam@example.com as data

### Jira Link 

https://themegrill.atlassian.net/browse/EVF-637

### Types of changes:


### Other information:



### Changelog entry

> Feature - Akismet integration.
